<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DrawingUi" name="currentFrame">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Current 
Frame
(PLACEHOLDER)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="changeColor">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>537</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="currectColorText">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>597</y>
      <width>95</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Current 
 Color:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentColorPreview">
    <property name="geometry">
     <rect>
      <x>697</x>
      <y>597</y>
      <width>95</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spritePreview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sprite Preview 
(PLACEHOLDER)</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="fpsSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>40</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Frame:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="frameSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>285</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>345</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>472</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraser">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>537</y>
      <width>80</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string>Eraser</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pencil">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>537</y>
      <width>80</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string>Pencil</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stamps">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>537</y>
      <width>120</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string>Stamps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearFrameButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>607</y>
      <width>132</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fillFrameButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>537</y>
      <width>132</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="Border">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>5</y>
      <width>522</width>
      <height>522</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Border</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="previewBorder">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>7</y>
      <width>206</width>
      <height>206</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Preview Border</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicateFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>250</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate Frame</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>537</y>
      <width>120</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>64 x 64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32 x 32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16 x 16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 x 8</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="previewSprite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>223</y>
      <width>250</width>
      <height>52</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Sprite</string>
    </property>
   </widget>
   <zorder>previewBorder</zorder>
   <zorder>Border</zorder>
   <zorder>currentFrame</zorder>
   <zorder>changeColor</zorder>
   <zorder>currectColorText</zorder>
   <zorder>currentColorPreview</zorder>
   <zorder>spritePreview</zorder>
   <zorder>fpsSlider</zorder>
   <zorder>label_5</zorder>
   <zorder>frameSpinBox</zorder>
   <zorder>addFrame</zorder>
   <zorder>deleteFrame</zorder>
   <zorder>eraser</zorder>
   <zorder>pencil</zorder>
   <zorder>stamps</zorder>
   <zorder>clearFrameButton</zorder>
   <zorder>fillFrameButton</zorder>
   <zorder>duplicateFrame</zorder>
   <zorder>sizeBox</zorder>
   <zorder>previewSprite</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingUi</class>
   <extends>QLabel</extends>
   <header>drawingui.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
